docs/API/InstrumentFormat.md

```diff
@@ -313,6 +333,104 @@ the user. It has the following form.
                score will be displayed with no accompagning text.
 
 
+### 2.1.6: CalcFieldElement
+
```

Comment:
This whole element type is unnecessary. That's what a score field type directly above it. All that needs to be done is have the Formula added as an option in the score type.